files/jquery.inview-fork/update.json

```diff
@@ -0,0 +1,9 @@
+
+{
+  "packageManager": "bower",
+  "name": "protonet/jquery.inview",
```

Comment:
The name is what is going to be used to search bower. `protonet/jquery.inview` is not a valid bower package name.
Either fix it or change to Github package manager